config/spotbugs-exclude.xml

```diff
@@ -404,4 +404,11 @@
     <Method name="getTargetFiles" />
     <Bug pattern="RCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE" />
   </Match>
+
+  <!-- Suppress DMC_DUBIOUS_MAP_COLLECTION for ConfigurationLoader.ID_TO_RESOURCE_NAME_MAP -->
```

Comment:
Comment is required to show link until what defects it should stay.
Not to repeat what is written in config itself